ManalHamdi/assignment1/ManalHamdi_As1_Q1.cc

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <ctype.h>
+using namespace std;
+
+bool isAnagramCaseSensitive(string s1, string s2) {
+	if (s1.size() != s2.size())
+		return false;
+
+	sort(s1.begin(), s1.end());
+	sort(s2.begin(), s2.end());
+
+	for (int i = 0; i < s1.size(); i++) {
+		if (s1[i] != s2[i])
+			return false;
+	}
+	return true;
+}
+bool isAnagramCaseInsensitive(string s1, string s2) {
+	if (s1.size() != s2.size())
+		return false;
+	for (int i = 0; i < s1.size(); i++) {
+		s1[i] = tolower(s1[i]);
+		s2[i] = tolower(s2[i]);
+	}
+	return isAnagramCaseSensitive(s1, s2);
+}
+bool isSentenceAnagram (string sent1, string sent2) {
+	vector <string> vSent1, vSent2;
+	if (sent1.size() != sent2.size())
```

Comment:
What if your inputs are `'aaa b'` and `'b   aaa'` (where the second string has three spaces between 'b' and 'aaa'). Shouldn't they be considered anagrams?